client/src/state/toasts/ToastsActions.js

```diff
@@ -0,0 +1,43 @@
+import ACTION_TYPES from './ToastsActionTypes';
+import { v4 as uuid } from 'uuid';
+
+const defaultOptions = {
+  inEffect: {opacity: 'show'},	// in effect
+  inEffectDuration: 600,				// in effect duration in miliseconds
+  stayTime: 3000,				// time in miliseconds before the item has to disappear
+  text: '',					// content of the item
+  stay: false,				// should the notice item stay or not?
+  type: 'notice' 			// could also be error, succes
+};
+
+/**
+ * Sets global config for the application.
+ *
+ * @param {object} config
+ */
+export function display(options) {
+  const key = 'toast-' + uuid();
+  const payload = { key, ...defaultOptions, ...options }
+
+  return (dispatch) => {
+    dispatch ({type: ACTION_TYPES.DISPLAY, payload});
+    if (!payload.stay) {
+      setTimeout(() => dismiss(key)(dispatch), payload.stayTime);
```

Comment:
There's some clever use of timeout here to trigger the fade out of the toast. It's not clear that it makes a lot of sense to have this here, but it has the advantage of keeping all the setTimeout logic in one place.